packages/concerto-cto/lib/printer.js

```diff
@@ -386,9 +386,16 @@ function toCTO(metaModel) {
             case `${MetaModelNamespace}.ImportAllFrom`:
                 result += `\nimport ${imp.namespace}.*`;
                 break;
-            case `${MetaModelNamespace}.ImportTypes`:
-                result += `\nimport ${imp.namespace}.{${imp.types.join(',')}}`;
+            case `${MetaModelNamespace}.ImportTypes`: {
+                const aliasedTypes = imp.aliasedTypes
+                    ? new Map(imp.aliasedTypes.map(({name,aliasedName})=>[name,aliasedName]))
+                    : new Map();
+                const commaSeparatedTypesString = imp.types.map((type) => aliasedTypes.has(type)
+                    ? `${type} as ${aliasedTypes.get(type)}`
+                    : type).join(',');
```

Comment:
did you run the formatter on this part? I think the trailing parentheses looks a bit off.
```suggestion
                const commaSeparatedTypesString = imp.types
                    .map(type => (aliasedTypes.has(type) ? `${type} as ${aliasedTypes.get(type)}` : type))
                    .join(',');
```

Reply:
```js
                const aliasedTypes = imp.aliasedTypes
                    ? new Map(
                        imp.aliasedTypes.map(({ name, aliasedName }) => [
                            name,
                            aliasedName,
                        ])
                    )
                    : new Map();
                const commaSeparatedTypesString = imp.types
                    .map((type) =>
                        aliasedTypes.has(type)
                            ? `${type} as ${aliasedTypes.get(type)}`
                            : type
                    )
                    .join(',');
```

Running formatted outputs as above. Made this change